conf/modules.config

```
@@ -55,8 +55,8 @@ process {
         ext.args = { 'test' in workflow.profile.tokenize(',') ?
                         // Additional configuration to speed processes up during testing.
                         // Note: BUSCO *must* see the double-quotes around the parameters
-                        '--tar --metaeuk --metaeuk_parameters \'"-s=2"\' --metaeuk_rerun_parameters \'"-s=2"\''
-                    : '--tar --metaeuk ' }
+                        '--tar '
+                    : '--tar ' }
```

This one needs to stay because it's what's used for actual pipeline runs.
```suggestion
                    : '--tar --metaeuk' }
```

I agree we'll move away from metaeuk, but that needs to be synchronised with BTK and that'll be in a next release